cmd/ocm/create/cluster/cmd.go

```diff
@@ -533,6 +546,14 @@ func preRun(cmd *cobra.Command, argv []string) error {
 	if isGcpMarketplace {
```

Comment:
We are doing the same check on line 529, I wonder if this can be refactored so that we only check once.

Reply:
we are checking if `isGcpMarketplace` twice, lines 529 and 548. I'm wondering if we could "merge" the two code blocks so that the logic is centralized.

Reply:
Merged